shared/js/toc.jsx

```diff
@@ -560,6 +563,70 @@ loop.shared.toc = (function(mozL10n) {
     }
   });
 
+  var RoomControlsView = React.createClass({
+    statics: {
+      OPEN_DELAY: 200,
+      CLOSE_DELAY: 350
+    },
+
+    propTypes: {
+      audio: React.PropTypes.object.isRequired,
+      dispatcher: React.PropTypes.instanceOf(loop.Dispatcher).isRequired,
+      participantStore: React.PropTypes.instanceOf(loop.store.ParticipantStore).isRequired,
+      roomState: React.PropTypes.string.isRequired,
+      screen: React.PropTypes.object.isRequired,
+      video: React.PropTypes.object.isRequired
+    },
+
+    getInitialState: function() {
+      return {
+        showRoomControls: this.props.roomState === ROOM_STATES.HAS_PARTICIPANTS
```

Comment:
Actually, it doesn't work because the room state is never updated if the webRTC connection isn't established. So, perhaps we should think about a different way to show the media controls. I tried with `activeRoomStore._hasParticipants` and it works for desktop but no for standalone because the room data collected from loop-server in guest don't contain the array of participants